project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.8
+sbt.version=0.13.7
```

Comment:
This wasn't necessary. I'll change it back. I was running into some errors, so I was trying to follow the scala.js turtorial to the letter in order to eliminate possible sources of error. Given that this was one, even if it seemed unlikely, I decided to temporarily revert. I should have made a note to myself to check it out again once I got things working.